tests/chainerx_tests/unit_tests/routines_tests/test_sorting.py

```diff
@@ -36,6 +36,34 @@
 ]
 
 
+_count_nonzero_params = [
```

Comment:
Could you add tests for `axis` is a tuple?

Reply:
Added.